qiskit/providers/aer/backends/unitary_simulator.py

```diff
@@ -169,16 +179,15 @@ def __init__(self,
 
         super().__init__(configuration,
                          properties=properties,
-                         available_methods=UnitarySimulator._AVAILABLE_METHODS,
                          provider=provider,
                          backend_options=backend_options)
 
     @classmethod
     def _default_options(cls):
         return Options(
             # Global options
-            shots=1024,
-            method="automatic",
```

Comment:
the method option is deprecated entirely for Unitary and StatevectorSimulators, since they are by definition a fixed method backend version of the AerSimulator